ytfzf

```diff
@@ -607,8 +644,7 @@ interface_thumbnails () {
 	sort_video_data_fn |
 	fzf -m \
 	--preview "sh $0 -U preview_img '"$thumbnail_viewer"' {} '"$video_json_file"'" \
-	--preview-window "left:50%:wrap" --layout=reverse |
-	trim_id > "$selected_id_file"
```

Comment:
how would the selection go into the id file here

Reply:
oh i see now, but doing it in the bottom case like this seems like it removes flexibility

Reply:
> oh i see now, but doing it in the bottom case like this seems like it removes flexibility

I thought about this as well, however if they need to do anything after returning the ids, they can put it in {} and redirect to /dev/null, but i can remove it.